src/main/config/proxy-compose.yaml.ctmpl

```diff
@@ -7,15 +7,11 @@ proxy:
   {{.Data.proxy_ports}}
   {{.Data.proxy_volumes}}
   environment:
-    AUTH_TYPE: AuthType None
-    AUTH_TYPE2: AuthType oauth20
     CALLBACK_URI: {{.Data.env_callback_uri}}
-    LOG_LEVEL: warn
-    OIDC_CLAIM: Require all granted
```

Comment:
The change from email to sub means that the REMOTE_USER environment variable will now have the numeric value of the google userid instead of the email.

The OIDC_CLAIM_email will ALWAYS have the email address stored in it.